.github/workflows/windows.yml

```diff
@@ -112,11 +112,10 @@ jobs:
 
       - name: Build
         run: |
-          # TODO: remove the build-dir and install-prefix flags once dev.py
-          # supplies meson with absolute paths. This is due to a pathlib bug
-          # on cp39-win.
-          python dev.py --build-dir=$pwd\build --install-prefix=$pwd\build-install build -j 2 --win-cp-openblas
-          meson install -C build
+          python dev.py build -j 2 --win-cp-openblas
+          # Necessary because GitHub Actions checks out the repo to D:\ while OpenBLAS
+          # got installed to C:\ higher up. The copying with `--win-cp-openblas` fails
+          # when things are split over drives.
           cp C:\opt\64\bin\*.dll $pwd\build-install\Lib\site-packages\scipy\.libs\
```

Comment:
Can you apply similar changes to L61-L70?

Reply:
Done, with a few more changes to keep the two jobs in sync. We may want to merge them later perhaps, they're almost the same.